flexget/ui/src/plugins/series/components/series-episode/series-episode.component.js

```diff
@@ -1,105 +1,79 @@
+                    show: vm.show,
+                    episode: vm.episode,
+                    releases: vm.releases
+                },
+                bindToController: true,
+                controllerAs: 'vm',
+                controller: function () {}
+            }
 
-      //Call from the page, to delete all releases
-      vm.deleteReleases = function() {
-        var confirm = $mdDialog.confirm()
-          .title('Confirm deleting releases.')
-          .htmlContent("Are you sure you want to delete all releases for <b>" + vm.episode.episode_identifier + "</b> from show " + vm.show + "?<br /> This also removes all seen releases for this episode!")
-          .ok("Forget")
-          .cancel("No");
+            $mdDialog.show(dialog);
 
-        $mdDialog.show(confirm).then(function() {
-          $http.delete('/api/series/' + $stateParams.id + '/episodes/' + vm.episode.episode_id + '/releases', { params: { forget: true}})
-            .success(function(data) {
-              //Remove all loaded releases from the page and set variables for the accordion
-              vm.releases = undefined;
-              vm.episode.episode_number_of_releases = 0;
-              releasesOpen = false;
-            })
-            .error(function(error) {
-              var errorDialog = $mdDialog.alert()
-                .title("Something went wrong")
-                .htmlContent("Oops, something went wrong when trying to forget <b>" + vm.episode.episode_identifier + "</b> from show " + vm.show + ":<br />" + error.message)
-                .ok("Ok");
+        }
 
-              $mdDialog.show(errorDialog);
-            });
-        });
-      }
+        //Call from the page, to delete all releases
+        vm.deleteReleases = function() {
+            var confirm = $mdDialog.confirm()
+            .title('Confirm deleting releases.')
+            .htmlContent("Are you sure you want to delete all releases for <b>" + vm.episode.episode_identifier + "</b> from show " + vm.show + "?<br /> This also removes all seen releases for this episode!")
+            .ok("Forget")
+            .cancel("No");
 
-      //Load the releases upon opening the accordion
-      function loadReleases() {
-        $http.get('/api/series/' + $stateParams.id + '/episodes/' + vm.episode.episode_id + '/releases')
-        .success(function(data) {
-          vm.releases = data.releases;
-        }).error(function(error) {
-          console.log(error);
-        });
-      }
+            $mdDialog.show(confirm).then(function() {
+                $http.delete('/api/series/' + vm.show.show_id + '/episodes/' + vm.episode.episode_id + '/releases', { params: { forget: true}})
+                .success(function(data) {
+                    //Remove all loaded releases from the page and set variables for the accordion
+                    vm.releases = undefined;
+                    vm.episode.episode_number_of_releases = 0;
+                    releasesOpen = false;
```

Comment:
A couple of these lines can be removed I think, they where used for the old behaviour to open the releases accordion.